src/lib/y2network/config_reader/sysconfig_dns.rb

```diff
@@ -74,6 +79,39 @@ def resolv_conf_policy
       #
       # @return [String]
       def hostname
```

Comment:
In yast/yast-yast2#921 I added a `Hostname` class and a reader only for the `Hostname`, just for cases where the complete `DNS` config is not needed at all but only the `hostname`. 

It is committed as [RFC].